base-merch-coin-example.md

```
@@ -0,0 +1,16 @@
+# Base Merch Coin Example
+
+Simple ERC20 Coin deployed on Base mainnet using thirdweb Coin template.
+
+## Contract Details
+- Address: https://thirdweb.com/base/0x47b8cBE61199b1260bBbD6760efe08FC2236c3d4
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)